boards/arm/sama5/sama5d3x-ek/Kconfig

```diff
@@ -180,8 +180,7 @@ endchoice # AT24 serial EPPROM configuration
 
 config SAMA5D3xEK_USBHOST_STACKSIZE
 	int "USB host waiter stack size"
-	default 1536 if USBHOST_HUB
-	default 1024 if !USBHOST_HUB
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.